### PR TITLE
Ingredient and process additions

### PR DESCRIPTION
**Pull request by aleene**

### CODE REVIEW

File: taxonomies/ingredients.txt [hunk 1]
```diff
@@ -9874,11 +9874,13 @@ sv:Färsk grädde
...
 ca:Crema agra, crema àcida
+da:syrnet fløde
 de:saure Sahne, Sauerrahm
+el:κρέμα γαλακτοσ
 eo:Acidkremo
-es:Crema agria, nata agria
+es:Crema agria, nata agria, nata ácida
 fa:خامه ترش
-fi:Hapankerma
+fi:Hapankerma, ranskankerma
```
Comment (Ban3) [inline taxonomies/ingredients.txt:9883]: Ranskankerma = Crème fraîche. I think it's already a synonym for that.

Comment (aleene) [reply taxonomies/ingredients.txt:9883]: Great, I just copied it from ingredient lists. So you see how well producers translate.

Comment (stephanegigandet) [reply taxonomies/ingredients.txt:9883]: So ", ranskankerma" should be removed, right?

Comment (Ban3) [reply taxonomies/ingredients.txt:9883]: @stephanegigandet Actually, I noticed that ranskankerma was under a wrong ingredient (`en:fresh-cream`) even before aleene's commit. I'll fix it by pushing to this branch.

Comment (stephanegigandet) [reply taxonomies/ingredients.txt:9883]: Perfect, thank you @Ban3 !

**Pull request merged**